tools/uzem/uzerom.h

```diff
@@ -49,9 +55,10 @@ struct RomHeader{
     uint8_t author[32];
     uint8_t icon[16*16];
     uint32_t crc32;
-    uint8_t mouse;
-	uint8_t description[64];
-    uint8_t reserved[114];
+    uint8_t psupport; //supported peripherals
+    uint8_t description[64];
+    uint8_t pdefault; //default enabled peripherals(Emulator only)
+    uint8_t reserved[113];
 };

```

Comment:
Please add a comment here that if this struct is modified, that it also needs to be modified in packrom.cpp